langchain4j-core/src/main/java/dev/langchain4j/model/input/Prompt.java

```diff
@@ -10,6 +10,9 @@
 import static dev.langchain4j.data.message.SystemMessage.systemMessage;
 import static dev.langchain4j.data.message.UserMessage.userMessage;
 
+/**
+ * Represents a prompt, typically produced by applying one or multiple values to a PromptTemplate.
```

Comment:
I would clarify the word prompt, like "prompt refers to the input provided by the user"